src/components/ImageView/index.js

```diff
@@ -233,16 +241,17 @@ class ImageView extends PureComponent {
                     <Image
                         source={{uri: this.props.url}}
                         isAuthTokenRequired={this.props.isAuthTokenRequired}
-                        style={this.state.zoomScale === 0 ? undefined : [
+
+                        // Hide image until finished loading to prevent showing preview with wrong dimensions.
+                        style={this.state.isLoading ? undefined : [
```

Comment:
[This minor issue](https://github.com/Expensify/App/issues/26882) was introduced by changes made in this PR.  The issue is fixed by this [PR](https://github.com/Expensify/App/pull/27739/files) by including `zoomScale` along with `isLoading`.